byterun/caml/misc.h

```diff
@@ -149,10 +149,10 @@ extern void caml_set_fields (intnat v, unsigned long, unsigned long);
 
 /* snprintf emulation for Win32 */
 
-#ifdef _WIN32
+#if defined(_WIN32) && defined(_MSC_VER) && _MSC_VER < 1900
```

Comment:
On Thu, Nov 19, 2015 at 3:43 AM, Alain Frisch <notifications@github.com>
wrote:

> For Mingw ports, I guess that _WIN32 is defined but not _MSC_VER -- is
> that right? Then this patch will disable the snprintf emulation for these
> ports. Is that ok?


The earlier code unconditionally applied the substitution for `snprintf`
when `defined(_WIN32)`.

​I don't have a Mingw install to hand but a quick google reveals that it
provides `snprintf`. So, if the substitution were no longer applied in the
context of Mingw, it should still work out.​ I thus appears the `defined
(_WIN32)` in that condition is superfluous.

-- 
Shayne Fletcher